### DISCUSSION TITLE
Create Windows MSI package for application

### DISCUSSION THREAD

**Discussion by Bekreth** [nicolas-graves/lfs-s3#61] [Q&A]
I came across this project recently and am needing to setup a bunch of windows machine for people unfamiliar with technology.  Wrapping this application in a windows MSI package will make download and install much easier, but adds a fair amount of overhead on the CI process. If this is something you're interested in having in this project, I was going to attach it similarly to how I've done so in this other project I oversee (https://github.com/Bekreth/jane_cli/tree/main/packaging)

**Comment by nicolas-graves** [comment]
As I've put in the README, I don't use Windows, but I'm willing to
include these kind of changes in case they help.

I think the easiest in this case is to go and fork the repo, do what you
would have done there, test it on your side (setup those windows
machines), and make a PR explaining (to a non-windows user) what the PR
allows (easy Windows install ?) and what the downsides are. I would be
glad to include it once I have an understanding of this.

Does that sound good to you?

Thanks!

-- 
Best regards,
Nicolas Graves
